CMake/add_project_kwiver.cmake

```diff
@@ -11,22 +11,24 @@ ExternalProject_Add(kwiver
   SOURCE_DIR kwiver
   BINARY_DIR kwiver-build
   STAMP_DIR ${DIVA_STAMP_DIR}
-  GIT_REPOSITORY "git://github.com/Kitware/kwiver.git"
-  GIT_TAG 3329915c323a2b09658e0fde15015805960a7d17
+  GIT_REPOSITORY "git://github.com/dmjoy/kwiver.git"
+  GIT_TAG 80ffce5bf75279aebffa4be7203f6c7c88cf2d6f
   CMAKE_CACHE_ARGS
-    -DBUILD_SHARED_LIBS:BOOL=ON
+    -DKWIVER_BUILD_SHARED:BOOL=${DIVA_BUILD_SHARED_LIBS}
+    -DBUILD_SHARED_LIBS:BOOL=${DIVA_BUILD_SHARED_LIBS}
+    -DKWIVER_BUILT_SHARED:BOOL=${DIVA_BUILD_SHARED_LIBS}
     -Dfletch_DIR:PATH=${fletch_DIR}
     -DKWIVER_ENABLE_ARROWS:BOOL=ON
     -DKWIVER_ENABLE_BURNOUT:BOOL=OFF
     -DKWIVER_ENABLE_CERES:BOOL=OFF
-    -DKWIVER_ENABLE_CUDA:BOOL=OFF
+    -DKWIVER_ENABLE_CUDA:BOOL=${DIVA_BUILD_WITH_CUDA}
```

Comment:
The intent of CUDA flag was to enable/disable optical flow in DIVA, based on my understanding of the CUDA arrow it is not related to DIVA at all

Reply:
Would enabling optical flow in DIVA (with this flag) require KWIVER to have CUDA enabled though?  If not, then I'll revert the change

Reply:
Not at the moment, although fletch has a similar flag fletch_BUILD_WITH_CUDA, its value would depend on DIVA_BUILD_WITH_CUDA, since it builds opencv's cuda package which is required by optical flow